.github/workflows/release-ios.yml

```
@@ -0,0 +1,150 @@
+          security set-keychain-settings -lut 21600 $KEYCHAIN_PATH
+          security unlock-keychain -p "$KEYCHAIN_PASSWORD" $KEYCHAIN_PATH
+          
+          # import certificate to keychain
+          security import $CERTIFICATE_PATH -P "$P12_PASSWORD" -A -t cert -f pkcs12 -k $KEYCHAIN_PATH
+          security set-key-partition-list -S apple-tool:,apple: -k "$KEYCHAIN_PASSWORD" $KEYCHAIN_PATH
+          security list-keychain -d user -s $KEYCHAIN_PATH
+      - name: Build native library
+        env:
+          IPHONEOS_DEPLOYMENT_TARGET: ${{ vars.IOS_DEPLOYMENT_TARGET || '14.0' }}
+          RUSTFLAGS: '-C link-arg=-Wl,-application_extension'
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:7: Double quote to prevent globbing and word splitting

(shellcheck)